src/condor_dagman/dagman_main.cpp

```diff
@@ -1282,6 +1287,22 @@ void condor_event_timer (int /* tid */) {
 		if (dagman.dag->GetDotFileUpdate()) { dagman.dag->DumpDotFile(); }
 	}
 
+	time_t printJobTableDelay = (time_t)dagman.config[conf::i::JobStateTableInterval];
+	if (printJobTableDelay && time(nullptr) - lastPrintJobTable >= printJobTableDelay) {
```

Comment:
Can we parenthesize this a bit more?  It's getting to be a bit much to go that far without any parens.